app/src/main/AndroidManifest.xml

```diff
@@ -11,22 +11,56 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Breadboard"
-        tools:targetApi="26"
         android:windowSoftInputMode="adjustResize"
         android:enableOnBackInvokedCallback="true">
         <activity
             android:name=".MainActivity"
+            android:taskAffinity="BreadBoard.MainActivity"
             android:exported="true"
-            android:label="@string/app_name"
             android:theme="@style/Theme.Breadboard">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-    </application>
-    <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+        <activity
+            android:name=".DeepLinkActivity"
+            android:taskAffinity="BreadBoard.DeepLinkActivity"
+            android:excludeFromRecents="true"
```

Comment:
Can you please remove this. It causes unexpected behaviour when using the android horizontal swipe on the gesture bar to switch between apps (it does not take you back to the app you'd expect) and I think there are reasons for wanting it to stay in recents regardless.